corehq/warehouse/models/staging.py

```diff
@@ -167,3 +177,70 @@ def raw_record_iter(cls, start_datetime, end_datetime):
         domain_ids = get_domain_ids_by_last_modified(start_datetime, end_datetime)
 
         return iter_docs(Domain.get_db(), domain_ids)
+
+
+class FormStagingTable(StagingTable):
+    slug = FORM_STAGING_SLUG
+
+    form_id = models.CharField(max_length=255, unique=True)
+
+    domain = models.CharField(max_length=255, default=None)
+    app_id = models.CharField(max_length=255, null=True)
+    xmlns = models.CharField(max_length=255, default=None)
+    user_id = models.CharField(max_length=255, null=True)
+
+    # The time at which the server has received the form
+    received_on = models.DateTimeField(db_index=True)
+
+    build_id = models.CharField(max_length=255, null=True)
+
+    # TODO: Should we be storing state or will reports only deal with Normal forms?
```

Comment:
I think this is a discussion point. For most reports we would only be considering 'normal' forms. There are some reports that look at other form types but they are not aggregated reports and so can probably be run off the raw data.